tests/virt_autotest/validate_system_health.pm

```diff
@@ -55,7 +55,7 @@ sub run {
 sub post_fail_hook {
     my $self = shift;
     diag("Module validate_system_health post fail hook starts.");
-    unless (defined(script_run("rm -f /root/{commands_history,commands_failure}", die_on_timeout => 0))) {
+    unless (defined(script_run("timeout 20 rm -f /root/{commands_history,commands_failure}"))) {
```

Comment:
one more place to use `reconnect_if_problematic`.

Reply:
I would like to not do too many changes at once. I only changed the script_run call here and leave the error handling in place, see the next lines. In the other file I extracted a function but it's not yet exported so not directly usable in this test module I guess. Can we keep it as is?

Reply:
I don't bother with extration, but I'd like test continue in the case of ssh console is broken. so let's change to use 'enter_cmd() + wait_serial` too?

Reply:
oh yes. I misunderstood my own code. `timeout 20 rm` would return 124 on timeout so the result of `script_run` would always be defined. Sorry about my confusion. Updated.